addon/templates/components/polaris-resource-list/item.hbs

```diff
@@ -79,11 +79,11 @@
       class="Polaris-ResourceList-Item__Actions"
       click=(action stopPropagation)
     }}
-      {{#polaris-button-group
-        segmented=(not persistActions)
-        data-test-id=(unless persistActions "shortcut-actions")
+      <PolarisButtonGroup
+        data-test-id={{unless persistActions "shortcut-actions"}}
```

Comment:
```suggestion
        data-test-id={{unless @persistActions "shortcut-actions"}}
```

?

Reply:
we will get to this component too shortly....this was just to get tests passing